### PR TITLE
ivy.collapse_repeated

### PR DESCRIPTION
**Pull request by Change-m**

### CODE REVIEW

**Comment by Change-m** [comment]
Close #5152

**Comment by Change-m** [comment]
Collapse_repeated function is belong to Neural Network which is wildly used in Tensorflow, implemented compositionally that's why it is in extension.py file.

File: ivy/functional/ivy/extensions.py [hunk 1]
```diff
@@ -453,3 +453,43 @@ def sinc(
...
     """
     return ivy.current_backend(x).sinc(x, out=out)
+
+
+@integer_arrays_to_float
+@to_native_arrays_and_back
+@handle_out_argument
+@handle_nestable
+@handle_exceptions
+def collapse_repeated(
```
Comment (zaeemansari70) [inline ivy/functional/ivy/extensions.py:463]: Only tensorflow has this function, it has to be implemented compositionally using other Ivy functions.

**Review by zaeemansari70 — CHANGES_REQUESTED** [review]
Your PR looks amazing ✨!
Just requested a few changes and left comments on them which needs to be addressed before we can merge your PR! 🙂

You have to come up with a logic to implement this function using ivy functions as implemented here, as only tensorflow has it and therefore it is to be implemented compositionally, https://lets-unify.ai/ivy/deep_dive/1_function_types.html#compositional-functions 🙂

Do let me know if you have any questions!
Thanks!

**Pull request closed without merging**